crates/fj-math/src/circle.rs

```diff
@@ -167,6 +167,75 @@ impl<const D: usize> approx::AbsDiffEq for Circle<D> {
     }
 }
 
+/// Calculated geometry that is useful when dealing with an arc
+pub struct ArcCircleData {
```

Comment:
Yeah, looking at it again, that makes sense. I initially imagined it to be a small addition to `Circle` but at this point it basically deserves its own module.

Took the chance to add some tests for it too, and discovered that angles with a magnitude greater than π radians were broken, so that's fixed now too :rocket:

Reply:
Thank you, and nice catch finding that bug!